README.md

```diff
@@ -49,9 +49,10 @@ If the Docker section above is unappealing, its easy to run mass.gov natively on
 ###### Notes
 - The site is browseable at https://mass.local
 - It takes a few minutes for the `dbmass` container start up.
-- [You may override ddev config locally](https://ddev.readthedocs.io/en/stable/users/extend/config_yaml/). For example, create a `.ddev/config.personal.yml` file and add a line `dbimage: massgov/mysql-sanitized:latest` to use our less sanitized DB.
-- Similarly, rename [.ddev/config.arm.yaml.example](https://github.com/massgov/openmass/blob/develop/.ddev/config.arm.yaml.example) to config.arm.yaml in order to use ARM containers suitable for the Apple M1 Macs.
+- [You may override ddev config locally](https://ddev.readthedocs.io/en/stable/users/extend/config_yaml/). create a `.ddev/config.personal.yml` file and add whatever you need.
+- Similarly, rename [.ddev/.env.example](https://github.com/massgov/openmass/blob/develop/.ddev/.env.example) to `.env` in order to use ARM containers suitable for the Apple M1 Macs. This is also how you specify the less sanitized variant of our database.
```

Comment:
Another option would be to publish the containers with the same name? Then docker should default to the currently running platform unless `--platform` is specified.

Home Assistant requires putting the architecture name into the image name (I think this pattern predates proper multi-platform support in docker), and it's a real pain with it being coded that way everywhere.

```console
docker run --rm -it --platform linux/arm64 alpine:latest uname -a
Linux cacc67861cb1 5.10.88-0-virt #1-Alpine SMP Tue, 28 Dec 2021 12:51:14 +0000 aarch64 Linux

docker run --rm -it --platform linux/amd64 alpine:latest uname -a
Unable to find image 'alpine:latest' locally
latest: Pulling from library/alpine
Digest: sha256:21a3deaa0d32a8057914f36584b5288d2e5ecc984380bc0118285c70fa8c9300
Status: Downloaded newer image for alpine:latest
Linux 2ab351d5684e 5.10.88-0-virt #1-Alpine SMP Tue, 28 Dec 2021 12:51:14 +0000 x86_64 Linux

Reply:
I looked at docs for making a multi-platform image and wasnt able to get very far. Happy to discuss that more.